src/coreclr/nativeaot/Runtime/eventpipe/ep-rt-aot.h

```diff
@@ -42,6 +46,10 @@
 #undef EP_ALIGN_UP
 #define EP_ALIGN_UP(val,align) _rt_aot_align_up(val,align)
 
+#ifdef TARGET_UNIX
+extern pthread_key_t eventpipe_tls_key;
```

Comment:
Can we use thread_local variable like CoreCLR instead? thread_local variables have better perf.

Reply:
We wanted to stay away from bringing in C++ runtime dependencies.

Reply:
The native AOT runtime uses `thread_local` and I do not see us changing that to `pthread_key`. It would leave a lot of perf on the table. Using `thread_local` would not bring any new dependencies.